### ISSUE TITLE
Disable RT throttling

### ISSUE THREAD

**Issue by shuhaowu** [ros-realtime/ros-realtime-rpi4-image#28]
See https://wiki.linuxfoundation.org/realtime/documentation/technical_basics/sched_rt_throttling.

Basically, the default `/proc/sys/kernel/sched_rt_runtime_us` means that there's a chance that the kernel would throttle/sleep a RT process once it uses up 950ms out of 1s of CPU time. This could result in deadline misses. The feature was implemented before the widespread availability of SMP and thus needed to avoid complete system lockup/denial-of-service. With multicore systems (especially with isolcpu), this shouldn't be needed.

Basically, we need a startup service that writes -1 to `/proc/sys/kernel/sched_rt_runtime_us`.

FYI: @carlossvg @LanderU

**Comment by carlossvg** [comment]
@shuhaowu I agree to disable it by default and add it to the configuration list in the documentation. I would put the script in the `rt-configuration-scripts` directory described in https://github.com/ros-realtime/ros-realtime-rpi4-image/issues/26
